modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -184,7 +184,7 @@ The products you need for this procedure are:
 | OS Version | Product Name
 | {redhat} 6 | RHEL6 Base x86_64
 | {redhat} 7 | RHEL7 Base x86_64
-| {redhat} 8 | RHEL or SLES ES or CentOS 8 Base
+| {redhat} 8 | RHEL or SLES ES
```

Comment:
The channel / product name does not change just because CentOS 8 is now unsupported

Reply:
reverted